### PR TITLE
4 based on extension

### PR DESCRIPTION
**Pull request by alackerbauer**
thanks again :)

### CODE REVIEW

File: auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java [hunk 1]
```diff
@@ -65,25 +66,47 @@ private void createAuditEvent(RequestDetails theRequestDetails, boolean wasSucce
...
+    PlanDefinition initiallyCreatedPlanDefinition;
+    if (allPlanDefinitions != null && !allPlanDefinitions.isEmpty() && allPlanDefinitions.size() > 0) {
+      initiallyCreatedPlanDefinition = (PlanDefinition) allPlanDefinitions.getResources(0, 1).get(0);
+    } else {
+      throw new ExceptionInInitializerError("Apparently no plandefinition has been created during initialization");
+    }
+
+    //add based on extension
+    Extension basedOnExtension = new Extension();
+    basedOnExtension.setUrl("http://aist.fh-hagenberg.at/fhir/extensions/auditevent-basedon-extension");
```
Comment (pointan) [inline auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java:80]: Is this similar to a xml namespace where just any url can occur or should this be our server url?

**Review by pointan — CHANGES_REQUESTED** [review]
Code look good, but please rebase locally first, there seems to be some conflicts.

File: auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java (hunk 1 above)
Comment (alackerbauer) [reply auditlog-fhir-server/src/main/java/ca/uhn/fhir/jpa/starter/MyConsentService.java:80]: yes, it is like a xml namespace that we are using here.

**Comment by alackerbauer** [comment]
because @pointan already approved the code (only the outstanding conflict needed to be resolved), I am merging myself now.

**Pull request merged**